src/providers/lock-screen/__tests__/lock-screen.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { TestBed } from '@angular/core/testing';
+import { LockScreenProvider } from '../lock-screen';
+import { LockScreenProviderMock } from '../__mocks__/lock-screen.mock';
+
+describe('Lock Screen Provider', () => {
+
+  let lockScreenProvider: LockScreenProvider;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [
+                { provide: LockScreenProvider, useClass: LockScreenProviderMock },
```

Comment:
Not much point in the test if you're mocking the thing your testing. You'll need to mock out the cordova plugin instead and use the real `LockScreenProvider`. 
Have a look at the `DeviceProvider` and its test for an example. You'll also need to create a mock for the cordova object in `karmaGlobalMocks.js`.